metadata_mapper/mappers/oai/contentdm/arck_mapper.py

```diff
@@ -0,0 +1,32 @@
+from ..contentdm_mapper import ContentdmRecord, ContentdmVernacular
+
+class ArckRecord(ContentdmRecord):
+    @staticmethod
+    def identifier_match():
+        return ["view/ARCK3D"]
+
+    def UCLDC_map(self):
+        return {
+            "source": self.source_metadata.get("source")
+        }
+
+    def map_is_shown_by(self):
+        values = self.source_metadata.get("relation")
+        if not values:
+            return
+
+        candidates = [v for v in values if "/thumbnail" in v]
+        if not candidates:
+            return
+
+        return candidates[-1]
+
+    def map_relation(self):
+        if self.map_is_shown_at():
```

Comment:
> I think this should be self.map_is_shown_by() - not self.map_is_shown_at()

That makes sense. Can do!

> I think maybe this is worth analyzing in the source metadata to see if we can simplify this logic, since there are only 11 collections that use this mapping.

Would you like me to do this analysis?

Reply:
Yes please! If you've put in a few hours and it's looking like a time suck though, let's check back in before you go any further?

Reply:
Ok, I'll review the volume of content to determine my approach. In the meantime, can we proceed with merging this method with the changes you suggested, and adding a TODO so we remember?